src/SFA.DAS.FindApprenticeship.Jobs/Infrastructure/Api/Responses/VacancyType.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace SFA.DAS.FindApprenticeship.Jobs.Infrastructure.Api.Responses;
+public enum VacancyType
+{
+    Apprenticeship = 0,
+    Traineeship = 1
```

Comment:
We _shouldnt_ have any but we do need to make sure that we arent indexing any traineeships

Reply:
I've had a look at this now and realised the `VacancyType` and `DisabilityConfident ` classes are not actually used - it seems these properties were removed from the outer API response and don't come through anymore. Do we need the outer API to either remove all traineeships from it's response, or the outer API to return the VacancyType property so that we can do this in jobs?

Reply:
update: have removed these 2 classes in the meantime